### PR TITLE
Add more logs for mapzen services

### PR DESCRIPTION
**Pull request by ethervoid**
@iriberri take a look please

### CODE REVIEW

**Comment by iriberri** [comment]
LGTM.

Not for now, but as a comment: Perhaps it could be useful to add also the fields for country and province and so on in the geocoder exceptions, for example for the scenario in which the geocoding could fail due to very poor input information so that we could know if those values were filled by the user or not.

**Comment by ethervoid** [comment]
Good idea. I've already done for all the log information, I missed to add it here :)

**Comment by ethervoid** [comment]
@iriberri recheck this please

File: server/lib/python/cartodb_services/cartodb_services/mapzen/geocoder.py [hunk 1]
```diff
@@ -19,14 +19,32 @@ def __init__(self, app_key, logger, base_url=BASE_URL):
...
+            else:
+                self._logger.error('Error trying to geocode using mapzen',
+                                data={"response_status": response.status_code,
+                                        "response_reason": response.reason,
+                                        "response_content": response.text,
+                                        "reponse_url": response.url,
+                                        "response_headers": response.headers,
+                                        "searchtext": searchtext,
+                                        "city": city, "country": country,
+                                        "state_province": state_province, })
```
Comment (iriberri) [inline server/lib/python/cartodb_services/cartodb_services/mapzen/geocoder.py]: Extra comma at the end of `state_province`

File: server/lib/python/cartodb_services/cartodb_services/mapzen/routing.py [hunk 2]
```diff
@@ -49,10 +49,14 @@ def calculate_route_point_to_point(self, waypoints, mode,
...
-                                     options})
+            self._logger.error('Error trying to calculate route using Mapzen',
+                               data={"response_status": response.status_code,
+                                     "response_reason": response.reason,
+                                     "response_content": response.text,
+                                     "reponse_url": response.url,
+                                     "response_headers": response.headers,
+                                     "waypoints": waypoints, "mode": mode,
+                                     "options": options})
             raise Exception('Error trying to calculate route using HERE')
```
Comment (iriberri) [inline server/lib/python/cartodb_services/cartodb_services/mapzen/routing.py]: It still says HERE here :)

**Comment by iriberri** [comment]
:eyes: :eyes:

**Comment by ethervoid** [comment]
Anything else?

**Comment by iriberri** [comment]
A ☕ , please.


Merge now!

**Pull request merged**